src/containers/SearchContainer/SearchContainer.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { FC, useCallback, useMemo, useState, useRef } from "react";
```

Comment:
I think this entire page is basically going to sit in the repo until legislation is done. Unfortunate but makes the most sense.